test/e2e/run-openrpc-api-test-coverage.ts

```diff
@@ -83,6 +83,11 @@ async function main() {
           new ConfirmationsRejectRule({
             driver,
             only: methodsWithConfirmations,
+            requiresEthAccountsPermission: [
+              'personal_sign',
+              'eth_signTypedData_v4',
+              'eth_getEncryptionPublicKey',
+            ]
```

Comment:
why doesn't this need to include `eth_sendTransaction`?

Reply:
It's not included upstream in the open rpc doc transform fn

https://github.com/MetaMask/metamask-extension/blob/jl/fix-api-specs-multichain/test/e2e/api-specs/transform.ts#L298-L305

Reply:
and I guess is there some reason we shouldn't just add fixtures for these permissions like we did on the multichain side and skip the additional permissions request flow in the tests here?

Reply:
Unsure tbh. Only goal is to get the wallet_invokeMethod checks working again in this PR